c10/core/impl/DeviceGuardImplInterface.h

```diff
@@ -301,6 +301,7 @@ struct NoOpDeviceGuardImpl final : public DeviceGuardImplInterface {
   }
   bool queryEvent(void* /*event*/) const override {
     TORCH_CHECK(false, D, " backend doesn't support events.")
+    return true;
```

Comment:
- Returning true is meaningless as it will be intercepted by TORCH_CHECK.

Reply:
removed.